neurodsp/filt/iir.py

```diff
@@ -39,8 +41,12 @@ def filter_signal_iir(sig, fs, pass_type, f_range, butterworth_order,
         If True, print out the transition and pass bandwidths.
     plot_properties : bool, optional, default: False
         If True, plot the properties of the filter, including frequency response and/or kernel.
+    save_properties : str
+        Path, including file name, to save filter properites to as a json.
```

Comment:
typo on 'properites'

Reply:
Fixed. I renamed this variable to `save_report` as suggested. I also changed the description to 'save a filter report'.